$MirOS: ports/lang/tcl/8.4/patches/patch-unix_tcl_m4,v 1.2 2008/02/25 11:15:50 bsiegert Exp $
--- unix/tcl.m4.orig	2008-02-06 16:38:26.000000000 +0100
+++ unix/tcl.m4	2010-01-12 23:07:31.000000000 +0100
@@ -1486,7 +1486,7 @@ dnl AC_CHECK_TOOL(AR, ar)
 	    UNSHARED_LIB_SUFFIX='${TCL_TRIM_DOTS}\$\{DBGX\}.a'
 	    TCL_LIB_VERSIONS_OK=nodots
 	    ;;
-	OpenBSD-*)
+	OpenBSD-*|MirBSD-*)
 	    case `arch -s` in
 	    m88k|vax)
 		SHLIB_CFLAGS=""
@@ -1561,6 +1561,9 @@ dnl AC_CHECK_TOOL(AR, ar)
 	    esac
 	    ;;
 	Darwin-*)
+	    UNSHARED_LIB_SUFFIX='${TCL_TRIM_DOTS}\$\{DBGX\}.a'
+	    TCL_LIB_VERSIONS_OK=nodots
+	    TCL_LIB_VERSIONS_OK=nodots
 	    CFLAGS_OPTIMIZE="-Os"
 	    SHLIB_CFLAGS="-fno-common"
 	    # To avoid discrepancies between what headers configure sees during
@@ -2091,7 +2094,7 @@ dnl # preprocessing tests use only CPPFL
 		    ;;
 		IRIX*)
 		    ;;
-		NetBSD-*|FreeBSD-*|OpenBSD-*)
+		NetBSD-*|FreeBSD-*|OpenBSD-*|MirBSD-*)
 		    ;;
 		Darwin-*)
 		    ;;
